package.json

```diff
@@ -590,6 +590,12 @@
 					"items": {
 						"type": "string"
 					}
+				},
+				"dart.selectDeviceOnConnect": {
+					"type": "boolean",
+					"default": true,
+					"description": "Select newly connected devices as current device.",
```

Comment:
Minor nit: I think prefixing with "Whether to..." would be more consistent with the descriptions of other options.